backends/module.mk

```diff
@@ -350,15 +350,6 @@ MODULE_OBJS += \
 	mixer/null/null-mixer.o
 endif
 
-ifeq ($(BACKEND),opendingux)
```

Comment:
I am not sure to understand why these lines are deleted.
I suspect this will break all other opendingux (beta) platforms.

Reply:
I had to do the same fix for RS90. Details are in commit message. Those files end up being included twice on opendingux. Once here and once because we declare opendingux as posix-compatible (since it is)